.rubocop.yml

```diff
@@ -46,3 +46,8 @@ Metrics/ParameterLists:
 
 Style/Lambda:
   Enabled: false
+
+# enable both hash-rocket and ruby19 hashmap syntax
+Style/HashSyntax:
+  Enabled: false
```

Comment:
Yea, [I just added it a couple months ago](https://github.com/bbatsov/rubocop/pull/3441). It's 0.43+ only.